addons/web/static/src/webclient/burger_menu/burger_menu.scss

```diff
@@ -125,15 +89,8 @@
         @include o-hover-text-color(rgba($-base-color, .8), $-base-color);
       }
 
-      .o_dropdown_active > span {
```

Comment:
I don't see any `o_dropdown_active` class on any element, am I missing something?

Reply:
Seems to be leftovers from when the burger-menu supported collapsable entries.
Ok to remove.